lnwallet/btcwallet/btcwallet.go

```diff
@@ -459,12 +459,39 @@ func (b *BtcWallet) ListAccounts(name string,
 //
 // This is a part of the WalletController interface.
 func (b *BtcWallet) ImportAccount(name string, accountPubKey *hdkeychain.ExtendedKey,
-	masterKeyFingerprint uint32, addrType *waddrmgr.AddressType) error {
+	masterKeyFingerprint uint32, addrType *waddrmgr.AddressType,
+	dryRun bool) (*waddrmgr.AccountProperties, []btcutil.Address,
+	[]btcutil.Address, error) {
 
-	_, err := b.wallet.ImportAccount(
-		name, accountPubKey, masterKeyFingerprint, addrType,
+	if !dryRun {
+		accountProps, err := b.wallet.ImportAccount(
+			name, accountPubKey, masterKeyFingerprint, addrType,
+		)
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		return accountProps, nil, nil, err
+	}
+
+	// Generate 5 addresses by default for the dry run.
+	accountProps, extAddrs, intAddrs, err := b.wallet.ImportAccountDryRun(
+		name, accountPubKey, masterKeyFingerprint, addrType, 5,
```

Comment:
Should move the `5` to a constant somewhere, likely "good enough" for now as a base value.